Content.Shared/Atmos/Rotting/AntiRottingOnBucleComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Shared.Atmos.Rotting;
+
+/// <summary>
+/// Perishable entities buckled to an entity with this component this will not rot.
+/// </summary>
+[RegisterComponent]
+public sealed partial class ActiveAntiRottingOnBuckleComponent : Component
```

Comment:
NetworkedComponent.

Reply:
I added it. Though I do not comprehend why clients may need its replication. My idea was to keep it server-side.

Reply:
@metalgearsloth, do I have to add `AutoGenerateComponentState` and `AutoNetworkedField` as well?

Reply:
no